messagebird/client.py

```diff
@@ -269,6 +269,11 @@ def conversation_create_webhook(self, webhook_create_request):
         return ConversationWebhook().load(
             self.request(CONVERSATION_WEB_HOOKS_PATH, 'POST', webhook_create_request, CONVERSATION_TYPE))
 
+    def conversation_update_webhook(self, id, update_request):
+        uri = CONVERSATION_WEB_HOOKS_PATH + '/' + str(id)
```

Comment:
Maybe it's convenient for the user to either provide docstrings here about `update_request` or point to the developer docs